saleor/graphql/discount/utils.py

```diff
@@ -28,27 +29,17 @@
 PREDICATE_OPERATOR_DATA_T = list[dict[str, Union[list, dict, str, bool]]]
 
 
+class PredicateObjectType(Enum):
+    CATALOGUE = "catalogue"
+    CHECKOUT = "checkout"
+    ORDER = "order"
```

Comment:
Do we use all of three? I might be wrong, but I think that we reduced it to two, right?

Reply:
Also, as we have `predicate` for catalogue, and for order, this enum should operate on `CATALOGUE` and `ORDER`. Is it correct?

Reply:
Those values are used for filtering. For `checkout` we are using `CheckoutDiscountedObjectWhere`, for `order` we will be using `OrderDiscountedObjectWhere`